onyo/main.py

```diff
@@ -44,14 +44,34 @@ def select_key_value(arg_values):
     return pairs
 
 
-def select_paths(arg_values):
+def select_paths(arg_values: list[str]):
```

Comment:
It is not to be kept. If we go on with the changes decided for `onyo new` it will be removed. Currently it is blocked by issue #287.

But I think these changes should be done separately, like discussed in the meeting.
(Probably adding the pyre thing should have been separately, too, but nobody is perfect)

Reply:
Sounds good to me. Resolving and we'll make it part of the #287 discussion.